zc_install/includes/languages/en_us/main.php

```diff
@@ -0,0 +1,1066 @@
+'TEXT_ADMIN_SETUP_USER_NAME' => 'Admin Superuser Name: ',
+'TEXT_EXAMPLE_USERNAME' => 'ie: bill',
+'TEXT_ADMIN_SETUP_USER_EMAIL' => 'Admin Superuser Email: ',
+'TEXT_EXAMPLE_EMAIL' => 'ie: my_email@example.com',
+'TEXT_ADMIN_SETUP_USER_EMAIL_REPEAT' => '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Retype email: ',
+'TEXT_ADMIN_SETUP_USER_PASSWORD' => 'Admin password: ',
+'TEXT_ADMIN_SETUP_USER_PASSWORD_HELP' => '<strong>REMEMBER THIS!!</strong>: Below is your initial temporary password for your Admin Superuser Account. Please ensure you make a note of it.',
+'TEXT_ADMIN_SETUP_ADMIN_DIRECTORY' => 'Admin Directory: ',
+'TEXT_ADMIN_SETUP_ADMIN_DIRECTORY_HELP_DEFAULT' => 'We were not able to change your admin directory automatically. You will need to change it yourself before you can access your Store Admin.',
+'TEXT_ADMIN_SETUP_ADMIN_DIRECTORY_HELP_NOT_ADMIN_CHANGED' => 'We did not change your admin directory automatically as it already seems to have been changed from the default.',
+'TEXT_ADMIN_SETUP_ADMIN_DIRECTORY_HELP_CHANGED' => 'Your store Admin directory may have been automatically renamed. Please ensure you make a note of the directory below.',
+'TEXT_ADMIN_SETUP_NEWSLETTER_SETTINGS' => 'Newsletter',
+'TEXT_ADMIN_SETUP_NEWSLETTER_EMAIL' => 'Newsletter Email: ',
+'TEXT_ADMIN_SETUP_NEWSLETTER_OPTIN' => 'Opt In: ',
+'TEXT_PAGE_HEADING_COMPLETION' => 'Setup Finished',
+'TEXT_COMPLETION_HEADER_MAIN' => '',
+'TEXT_COMPLETION_INSTALL_COMPLETE' => 'One more thing!',
+'TEXT_COMPLETION_INSTALL_LINKS_BELOW' => 'The links to access your storefront and admin dashboard are shown below:',
+'TEXT_COMPLETION_UPGRADE_COMPLETE' => 'Congratulations, your upgrade is now complete.',
+'TEXT_COMPLETION_ADMIN_DIRECTORY_WARNING' => 'Your admin directory could not be renamed automatically, you will need to rename your admin directory before accessing it',
+'TEXT_COMPLETION_INSTALLATION_DIRECTORY_WARNING' => 'Please delete the "zc_install" folder now',
+'TEXT_COMPLETION_INSTALLATION_DIRECTORY_EXPLANATION' => 'To prevent unauthorized persons from overwriting your installation, administering your installation will be disabled until the folder has been deleted.',
+'TEXT_COMPLETION_CATALOG_LINK_TEXT' => 'Your Storefront',
+'TEXT_COMPLETION_ADMIN_LINK_TEXT' => 'Your Admin Dashboard',
+'TEXT_PAGE_HEADING_DATABASE_UPGRADE' => 'Database Upgrade',
+'TEXT_DATABASE_UPGRADE_HEADER_MAIN' => '',
+'TEXT_DATABASE_UPGRADE_STEPS_DETECTED' => 'The following list shows the various upgrade steps we detected are required for your database.',
+'TEXT_DATABASE_UPGRADE_LEGEND_UPGRADE_STEPS' => 'Please confirm your desired upgrade steps',
+'TEXT_DATABASE_UPGRADE_ADMIN_CREDENTIALS' => 'Admin Credentials (SuperUser)',
+'TEXT_VALIDATION_ADMIN_CREDENTIALS' => 'To authorize the database upgrade, you must enter an admin username and password with SuperUser permissions in your store.',
+'TEXT_HELP_TITLE_UPGRADEADMINNAME' => TEXT_DATABASE_UPGRADE_ADMIN_CREDENTIALS,
```

Comment:
`TEXT_DATABASE_UPGRADE_ADMIN_CREDENTIALS` refers to `'TEXT_DATABASE_UPGRADE_ADMIN_CREDENTIALS' => 'Admin Credentials (SuperUser)',` a couple lines above, but that constant won't be defined anymore!